indico/web/templates/base.html

```diff
@@ -11,14 +11,20 @@
     <script type="text/javascript" src="{{ url_for('assets.js_vars_global') }}"></script>
 
     <script>
-        window.CKEDITOR_BASEPATH = {{ url_for('assets.js', filename='lib/ckeditor/')|tojson() }};
+        window.CKEDITOR_BASEPATH = {{ url_for('assets.dist', filename='js/ckeditor/')|tojson() }};
     </script>
 
+    <!-- Webpack bundles -->
+        {{ webpack['common.js'] }}
+        {{ webpack['main.js'] }}
+        {{ webpack['main.css'] }}
+    <!-- end -->
```

Comment:
is that comment (and the indentation of the last three lines) really useful? ;)

Reply:
The comment was actually quite useful while moving to Webpack, I agree now it doesn't make much sense.